bigquery-antipattern-recognition/src/main/java/com/google/zetasql/toolkit/antipattern/cmd/AntiPatternCommandParser.java

```diff
@@ -141,10 +152,29 @@ public Options getOptions() {
             Option.builder(REWRITE_SQL_FLAG_NAME)
                     .argName(REWRITE_SQL_FLAG_NAME)
                     .required(false)
-                    .desc("flag specifying if the queries should be rwwritten using an LLM")
+                    .desc("flag specifying if the queries should be rewritten using an LLM")
                     .build();
     options.addOption(rewriteSQLFlag);
 
+    Option llmStrictValidation =
+            Option.builder(LLM_STRICT_VALIDATION_FLAG_NAME)
+                    .argName(LLM_STRICT_VALIDATION_FLAG_NAME)
+                    .required(false)
+                    .desc("flag specifying that LLM results will be discarded if the validation fails (Syntactically " +
+                            "invalid or antipattern still present)")
+                    .build();
+    options.addOption(llmStrictValidation);
+
+    Option llmRetriesSQLFlag =
+            Option.builder(LLM_RETRIES_FLAG_NAME)
+                    .argName(LLM_RETRIES_FLAG_NAME)
+                    .hasArg()
```

Comment:
nit. if it has args it should not be called a flag

Reply:
Will remove the flag part

Reply:
Done